tests/wake-format/basic/stdout

```diff
@@ -717,3 +706,16 @@ export tuple a +++ b =
 export tuple (z: Banger) =
     export X: Integer
 
+export data List a =
+    # The empty list. Nil represents a list with no elements.
+    Nil
+    # The prepend constructor. This is often called "cons".
+    #
+    # Examples:
+    # ```
+    #   10, list       # Add a number to `list`
+    #   1, 2, 3, Nil   # Create a list of 3 elements
+    # ```
+    (head: a), (tail: List a) # comment
+
+
```

Comment:
I'm sure you've got it on a list somewhere, but we'll probably want to figure out avoiding additional lines at the end of files.  Doesn't look like it's (purely) related to this PR, though, so that's probably something for future work.

Reply:
yeah, its something I've been thinking about. I don't have a solution yet but it'll be taken care of